index.html

```diff
@@ -12,8 +12,24 @@
 </head>
 
```

Comment:
Your index.html file is coming along great!  The layout is almost complete!  See my comment in the about.html, it will also help you here to complete the layout.